inc/GameMatch.h

```diff
@@ -0,0 +1,35 @@
+//
+// Created by danilo.bezerra on 12/12/2023.
+//
+
+#ifndef UNO_GAME_GAMEMATCH_H
+#define UNO_GAME_GAMEMATCH_H
+
+#include <memory>
+
+#include "Deck.h"
+#include "Player.h"
+
+class GameMatch {
+    Deck deck;
+
+    std::vector<std::unique_ptr<Player>> players;
+    std::vector<Card> drawPile;
+    std::vector<Card> discardPile;
```

Comment:
Wouldn't it be better having a vector to shared ptr Card? From looking at this, it would seem cards will be copied somewhere

Reply:
Yeah, I need to think about this one. I believe the idea would to move the cards from the deck to draw pile and eventually to the discard pile. So I didn't stop to refactor this one yet.